### ISSUE TITLE
average_position_price не содержит НКД для облигаций.

### ISSUE THREAD

**Issue by Mikhail-Smirnoff** [TinkoffCreditSystems/invest-openapi#644]
С недавних пор average_position_price перестала содержать текущий НКД для облигаций. Т.е. сейчас average_position_price и average_position_price_no_nkd выдают абсолютно одинаковые данные и НКД нет возможности выяснить и учесть. 

![nkd](https://user-images.githubusercontent.com/52107326/146093950-d154d963-6a96-44d6-accf-880dbf45b7ef.png)

**Comment by vrubanov** [comment]
Да, у меня так же некорректно работает - НКД перестал учитываться...

**Comment by HiHat** [comment]
У НКД печальная судьба в API.
Сначала цена была только с НКД, потом добавили average_position_price_no_nkd, а теперь average_position_price без НКД стал и НКД нельзя узнать!
Вернулись к истокам :(
